src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,16 +1,196 @@
+            onChange={(event) => {
+              setTitle(event.target.value);
+            }}
+            onClick={() => {
+              validateField(title, setTitleError, 'title');
+            }}
+          />
+          <p className="form__error">{!title && titleError}</p>
+        </div>
+
+        <div className="form__unit">
+          <input
+            type="text"
+            name="description"
+            placeholder="description"
+            className={classnames('input form__input', {
+              'is-primary': description,
+            })}
+            value={description}
+            onChange={(event) => {
+              setDescription(event.target.value);
+            }}
+          />
+        </div>
+
+        <div className="form__unit">
+          <input
+            type="text"
+            name="imdbId"
+            placeholder="id (must have 9 symbols)"
+            className={classnames('input form__input', {
+              'is-danger': (!imdbId && imdbIdError)
+              || (imdbIdError && checkRepeatedId),
+              'is-primary': imdbId,
+            })}
+            value={imdbId}
+            onChange={(event) => {
+              setImdbId(controlValidId(event.target.value));
+
+              if (checkRepeatedId) {
+                setImdbIdError('Use another id please');
+              } else {
+                setImdbIdError('');
+              }
+            }}
+            onClick={() => {
+              validateField(imdbId, setImdbIdError, 'id');
+            }}
+          />
+          <p className="form__error">
+            {(!imdbId && imdbIdError) || (checkRepeatedId && imdbIdError)}
+          </p>
+        </div>
+
+        <div className="form__unit">
+          <input
+            type="text"
+            name="imgUrl"
+            placeholder="image url"
+            value={imgUrl}
+            className={classnames('input form__input', {
+              'is-danger': !imgUrl && imgUrlError,
+              'is-primary': imgUrl,
+            })}
+            onChange={(event) => {
+              setImgUrl(event.target.value);
+            }}
+            onClick={() => {
+              validateField(imgUrl, setImgUrlError, 'image URL');
+            }}
+          />
+          <p className="form__error">{!imgUrl && imgUrlError}</p>
+        </div>
+
+        <div className="form__unit">
+          <input
+            type="text"
+            name="imdbUrl"
+            placeholder="imbd url"
+            className={classnames('input form__input', {
+              'is-danger': !imdbUrl && imdbUrlError,
+              'is-primary': imdbUrl,
+            })}
+            value={imdbUrl}
+            onChange={(event) => {
+              setImdbUrl(event.target.value);
+            }}
+            onClick={() => {
+              validateField(imdbUrl, setImdbUrlError, 'imdb URL');
+            }}
+          />
+          <p className="form__error">{!imdbUrl && imdbUrlError}</p>
```

Comment:
wrap whole element with condition, you don't need blank `p` element on your page, fix it everywhere
```suggestion
          {!imdbUrl  && (
            <p className="form__error">{imdbUrlError}</p>
          )}
```